DirectProgramming/DPC++FPGA/Tutorials/GettingStarted/fpga_template/src/CMakeLists.txt

```diff
@@ -0,0 +1,104 @@
+###############################################################################
+### Customize these build variables
+###############################################################################
+set(SOURCE_FILE fpga_template.cpp)
+set(TARGET_NAME fpga_template)
+
+# When targeting a board, the `FPGA_BOARD` will be targeted. Options are:
+#   intel_s10sx_pac:pac_s10
+#   intel_s10sx_pac:pac_s10_usm
+#   intel_a10gx_pac:pac_a10
+set(FPGA_BOARD "intel_s10sx_pac:pac_s10_usm")
+
+set(USER_HARDWARE_FLAGS "")
+# use cmake -DUSER_HARDWARE_FLAGS=<flags> to set extra flags for FPGA backend
+# compilation
+
+set(USER_FLAGS "")
+# use cmake -DUSER_FLAGS=<flags> to set extra flags for general compilation
+
+###############################################################################
+### no changes after here
+###############################################################################
+
+# define the different build targets the generated build script will support.
+set(EMULATOR_TARGET ${TARGET_NAME}.fpga_emu)
+
+# Full-system
+set(BOARD_REPORT_TARGET ${TARGET_NAME}.fpga_report)
+set(BOARD_SIMULATOR_TARGET ${TARGET_NAME}.fpga_sim)
+set(BOARD_TARGET ${TARGET_NAME}.fpga)
+
+# FPGA board selection
+if(DEFINED FPGA_BOARD)
+    message(STATUS "Configuring the design to run on FPGA accelerator board ${FPGA_BOARD}")
+endif()
+
+# This is a Windows-specific flag that enables exception handling in host code
+if(WIN32)
+    set(WIN_FLAG "/EHsc")
+    # add qactypes to link command on Windows only
+    set(QACTYPES_LINK "-qactypes")
```

Comment:
I don't think we need actypes for this basic tutorial

Reply:
disagree: qactypes should be included as boilerplate because it's terrible UX to make users add it in. This design is supposed to serve as a template, so I want -qactypes to stay.

Reply:
Interesting - should we push Ajay's team to make this happen rather than complicating our build files?

Reply:
I've [tried](https://hsdes.intel.com/appstore/article/#/18023990759) that, but they are reluctant because it might cause confusion between FPGA/GPU/CPU targets.

Reply:
We already have an `-fintelfpga`, this should be embedded in that (which would eliminate the discussion about FPGA/GPU/CPU confusion :)

Reply:
agreed. for now I will still leave `-qactypes` until they agree to make it part of `-fintelfpga`.